src/validators/doors.rs

```diff
@@ -20,14 +18,15 @@ impl Handler<'_> for DoorValidator {
         _: &Context,
         record: &Cell,
         reference: &Reference,
-        id: &String,
-        _: &Vec<&Reference>,
+        id: &str,
+        _: &[&Reference],
         _: usize,
     ) {
-        if id == "prisonmarker" && reference.door_destination_cell.is_none() {
+        // TODO: G7 verify this makes sense
```

Comment:
In what sense?

Reply:
I had some other code here and forgot to remove this comment. Don't remember exactly what I was on about, but the logic looks fine to me as is.